Kitodo/src/main/java/de/sub/goobi/forms/SpracheForm.java

```diff
@@ -135,6 +161,15 @@ public void switchLanguage(String langCodeCombined) throws IOException {
         } else {
             locale = new Locale(languageCode[0]);
         }
+        try {
+            User user = serviceManager.getUserService().getById(serviceManager.getUserService().getAuthenticatedUser().getId());
+            user.setLanguage(locale.toString());
+            serviceManager.getUserService().save(user);
+        } catch (DataException e) {
+            logger.error("Error in saving user : " + e.getMessage());
+        } catch (DAOException e) {
+            logger.error("Error in retrieving user" + e.getMessage());
```

Comment:
In other places users get informed that there is an error. Why not here?

Reply:
Yes, I did.
It 's not too long according to the CheckStyle.